medusa/post_processor.py

```diff
@@ -1255,7 +1272,7 @@ def process(self):
         # generate nfo/tbn
         try:
             # Reset the indexer_api for this series, as we don't want to use 'old' cached (to the series's indexer_api attribute) content.
-            ep_obj.series.indexer_api = None
+            self.clear_indexer_api(ep_obj)
```

Comment:
Reset the cached indexer_api for the series. But only do it once. Meaning we get the series data from the indexer once during a post process action.